cmd/admin.go

```diff
@@ -8,7 +8,7 @@ import (
 	"time"
 
 	"github.com/jmoiron/sqlx/types"
-	"github.com/labstack/echo/v4"
+	"github.com/labstack/echo"
```

Comment:
I don't think this should be changed.

> Any of these versions will allow you to import Echo as github.com/labstack/echo/v4 which is the recommended way of using Echo going forward.

https://github.com/labstack/echo

Reply:
Fixed, my package woes was caused by some bad copy-pasta. :spaghetti: